validators/valset/types.go

```diff
@@ -0,0 +1,111 @@
+package valset
+
+import (
+	"fmt"
+
+	"github.com/0xPolygon/polygon-edge/consensus/ibft/signer"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/0xPolygon/polygon-edge/validators"
+)
+
+type SignerGetter func(uint64) (signer.Signer, error)
+
+type ValidatorTypeGetter func(uint64) (validators.ValidatorType, error)
+
+// Define the type of the validator set
+type SourceType string
+
+const (
+	// For validators saved in-memory
+	Snapshot SourceType = "Snapshot"
+
+	// For validators managed in contract
+	Contract SourceType = "Contract"
+)
+
+var sourceTypes = map[string]SourceType{
```

Comment:
This map and the related `ParseSourceType` seem to be unused

Reply:
+1

Reply:
Removed c3d08bc5d4f5b33ccdef73617312fc55c8b16463